### ISSUE TITLE
error: name code chunk l.75 when rendering MS.qmd

### ISSUE THREAD

**Issue by MalikaIhle** [lmu-osc/code-publishing#25]
as reported by @anjabetz

**Comment by fkohrt** [comment]
@anjabetz Can you provide me with more context about this error message?

That is, during which step in the tutorial did you encounter the message? Can you share the article file `MS.qmd` with me?

**Comment by anjabetz** [comment]
Of course: The first time I tried rendering, I got the error message "Quitting from lines 78-87 [unnamed-chunk-2] (Manuscript.qmd)". I added a random label to the chunk in question and the rendering was successful.

**Comment by fkohrt** [comment]
Thank you! During which step in the tutorial did you encounter the message? Can you share the file `MS.qmd` with me by adding it within a message in this issue?

**Comment by MalikaIhle** [comment]
My understanding is that she is talking about the manuscript file provided within the tutorial in the project set up page. I don’t recall whether or not I had an error but I have a vague memory of an error which didn’t not affect however the building of the pdf.

**Comment by fkohrt** [comment]
When I follow the tutorial and render the manuscript file provided with the tutorial, I don't get any error message. So the first thing would be to check whether it is really the same file or there are some differences.

**Comment by fkohrt** [comment]
After the rewrite in 396e2b6, I think it makes sense to close this issue and see if anything resurfaces.
